### ISSUE TITLE
I was waiting the move of my opponent but when HIS time reach the zero i lost the game.. it turns out that he was waiting my move. Idk what happened.   Redmi note8, android 10, in app

### ISSUE THREAD

**Issue by Bobo1899** [lichess-org/lichobile#1522]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by richmartin** [comment]
A similar thing just happened to me. Playing a Classical game and after waiting a long time for my opponent to move, he messaged me in chat saying "please move!". I came out of the game and went back in again to see that I had lost about 10 minutes and it was now my move.

Have there been any recent changes in how the app acknowledges receipt of a new move?

**Comment by ZunderCodes** [comment]
I had this happen to me at the time Lichess restarted

**Comment by AnuarAssylbekov** [comment]
Same here. I thought it was weird for my opponent to think for that long. So I made this screenshot.
![Screenshot_20210206-142939_lichess](https://user-images.githubusercontent.com/78639606/107113487-6fc63100-6889-11eb-85c3-5a9ebc34f7b2.jpg)
Then it turned out that I lost.
![Screenshot_20210206-144202_lichess](https://user-images.githubusercontent.com/78639606/107113529-b3209f80-6889-11eb-8110-61ed20cae15b.jpg)

**Comment by oopsmouseslip** [comment]
This has happened to me twice.
Twice, it was lichess restarting. This is when the bug happens. In one game I lost 6 rapid points, in another I gained 11 classical points. 
1 idea would be to have games going on during the restart not affect rating.

**Comment by kolorado** [comment]
I can also confirm this. I don't think it was related to system restart though. I played on the Android app (on the 6th of February), and I had about 40 secs on my clock, my opponent about 3 mins. I had mate in 1, my opponent had a single legal move, but didn't make that move.  His status remained green. After a while, I tried to send "gg" to gently suggest to make a move or to resign, but the message didn't seem to get through. His time was running down, but the dot showing his status went grey. I was surprised not to see the usual "your opponent left the game" message. Eventually, his time ran out, but I didn't get the points. When I went back to the game, I saw that he had made the only legal move (when??) and my time ran out (I didn't premove to mate). That was very frustrating...
